Sources/ProjectDescription/FileList.swift

```diff
@@ -4,15 +4,21 @@ public struct FileList: Codable, Equatable {
     /// List glob patterns.
     public let globs: [Path]
 
+    /// Relative glob patterns for excluded files.
+    public let excluding: [Path]
```

Comment:
I think for Sources and Resources, the excluding is relative to the glob, and I guess we want to have the same here, right @pepicrft / @fortmarek ?

Reply:
yes, we definitely want to keep the behavior consistent

Reply:
They have special subtype for this: SourceFileList -> SourceFileGlob,
ResourceFileElements -> ResourceFileElement

Here we have only FileList, so I assume that for quick solution (as it's anyway is optional param and to reduce API significant changes) - leave FileList for that and extract it to new subtype on the real demand

Reply:
That's a good point - I'd be for creating a special type for this, though. It should be possible to make it backwards-compatible, I believe.

Reply:
@fortmarek ok, let's do it, which name is better - FileGlob or HeaderFileGlob
I suggest use HeaderFileGlob, as seems like FileList is used in Headers only and it's better to make usage more clear
(and later, witch major version change we can rename FileList to HeaderFileList and these classes will be convenient with SourceFileList/SourceFileGlob)